src/TaskManager.java

```diff
@@ -0,0 +1,38 @@
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class TaskManager {
+    private final ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public TaskManager() {
+
+    }
+
+    public void add(Task task) {
+        this.tasks.add(task);
+    }
+
+    public boolean isStartWithD(String name) {
+        return name.charAt(0) == 'D';
+    }
```

Comment:
this func should not be in this class do in the filter
getName.chatAt(0) == 'D'